mmdt/polls/admin.py

```diff
@@ -7,6 +9,22 @@ class ChoiceInline(admin.TabularInline):
     model = Choice
     extra = 3
 
+def export_to_csv(modeladmin, request, queryset):
+    response = HttpResponse(content_type = 'text/csv')
+    response['Content-Disposition'] = 'attachment; filename="poll_results.csv"'
+    writer = csv.writer(response)
+
+    # Write the headers to csv file
+    writer.writerow(['Question', 'Choice', 'Votes'])
+
+    # Write data
+    for question in queryset:
+        for choice in question.choice_set.all():
+            writer.writerow([question.question_text, choice.choice_text, choice.votes])
+    return response
+
+export_to_csv.short_description = "Export Selected Questions to CSV"
```

Comment:
Let's try to be consistent. Python allows both double quotes and single quotes for strings in the codebase, but it is better for us to be strict with one way. It seems like other file names are written in double quotes, so we should decide which one we should choose. Let's confirm with other codebases and synchronize with them.